src/components/Ui/Input.vue

```diff
@@ -21,14 +25,13 @@ function handleInput(e) {
 </script>
 
 <template>
-  <UiButton
-    class="text-left width-full mb-2 d-flex px-3"
-    :class="{ 'border-red': error }"
-  >
+  <UiButton class="text-left width-full mb-2 d-flex px-3" :class="{ 'border-red': error }">
```

Comment:
Any idea why your linter change these?

Reply:
I add multiple extention, that maybe not correct for the project, I disable them now and use prettier seems it's the same as yours now.

Reply:
@NftTopBest still there are a lot of lint differences in this PR.

Reply:
You should be able to fix them by doing `yarn` and `yarn lint` no?

Reply:
> You should be able to fix them by doing `yarn` and `yarn lint` no?

I try yarn lint, the code result don't have change ;)

Reply:
just push a new format code, pls check
I disable other format plugin and install vetur.

Reply:
@samuveth